Makefile

```diff
@@ -41,7 +41,7 @@ else
 	REGISTRY = $(RP_IMAGE_ACR)
 endif
 
-RP_IMAGE_LOCAL ?= localhost/aro
+RP_IMAGE_LOCAL ?= aro
```

Comment:
Sure, this can be verified with a pull of master and running `make ci-rp` without caching - I'm using podman 4.9.3.

It's an error which happens during the first `RUN` clause of the first `--cache-from` build, which ultimately resolves, but extends the build duration and causes confusing errors unnecessarily. It appears that when using `--cache-from`, with an image name with a well-defined domain (even `localhost`), podman first assumes that images are coming from an HTTPS server running an image repository, rather than local cache (perhaps there's an implicit `podman image pull` on that image?). Here's the errors:

```
[2/2] STEP 8/23: RUN curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b /usr/local/bin v1.56.2 &&     golangci-lint --version || (echo "golangci-lint not found" &&
exit 1)
WARN[0001] Failed, retrying in 2s ... (1/3). Error: initializing source docker://localhost/aro-portal-build:e47e553e08dcfb8b8e96b79af98d39c497f3ad4d0002116adac0c5b3420cb906: pinging container registry localhost: G
et "https://localhost/v2/": dial tcp [::1]:443: connect: connection refused
WARN[0003] Failed, retrying in 2s ... (2/3). Error: initializing source docker://localhost/aro-portal-build:e47e553e08dcfb8b8e96b79af98d39c497f3ad4d0002116adac0c5b3420cb906: pinging container registry localhost: G
et "https://localhost/v2/": dial tcp [::1]:443: connect: connection refused
WARN[0005] Failed, retrying in 2s ... (3/3). Error: initializing source docker://localhost/aro-portal-build:e47e553e08dcfb8b8e96b79af98d39c497f3ad4d0002116adac0c5b3420cb906: pinging container registry localhost: G
et "https://localhost/v2/": dial tcp [::1]:443: connect: connection refused
golangci/golangci-lint info checking GitHub for tag 'v1.56.2'
golangci/golangci-lint info found version: 1.56.2 for v1.56.2/linux/amd64
golangci/golangci-lint info installed /usr/local/bin/golangci-lint
golangci-lint has version 1.56.2 built with go1.22.0 from 58a724a0 on 2024-02-15T18:01:51Z
WARN[0009] Failed, retrying in 2s ... (1/3). Error: initializing source docker://localhost/aro-portal-build:e47e553e08dcfb8b8e96b79af98d39c497f3ad4d0002116adac0c5b3420cb906: pinging container registry localhost: G
et "https://localhost/v2/": dial tcp [::1]:443: connect: connection refused
WARN[0011] Failed, retrying in 2s ... (2/3). Error: initializing source docker://localhost/aro-portal-build:e47e553e08dcfb8b8e96b79af98d39c497f3ad4d0002116adac0c5b3420cb906: pinging container registry localhost: G
et "https://localhost/v2/": dial tcp [::1]:443: connect: connection refused
WARN[0013] Failed, retrying in 2s ... (3/3). Error: initializing source docker://localhost/aro-portal-build:e47e553e08dcfb8b8e96b79af98d39c497f3ad4d0002116adac0c5b3420cb906: pinging container registry localhost: G
et "https://localhost/v2/": dial tcp [::1]:443: connect: connection refused
```

When using non-domained names (e.g. `aro`), podman knows to hit the local cache (not served on HTTPS) and these errors go away - even though the name is the same and it's ultimately tagged as `localhost/aro` in the local cache:

```
$ podman image list
REPOSITORY                                   TAG                    IMAGE ID      CREATED         SIZE
localhost/aro                                latest                 ef8293611aea  42 seconds ago  520 MB
localhost/aro-builder                        latest                 2a3ba8c14d03  55 seconds ago  3.76 GB
localhost/aro-portal-build                   latest                 e9cfbac079d9  8 minutes ago   1.09 GB
```

Reply:
Thanks, ship it!